docs/patterns/buttons.md

```diff
@@ -84,6 +84,11 @@ To import just this component into your project, copy the snippet below and incl
 
 ```scss
 @import 'patterns_buttons';
+@include vf-p-buttons;
+
+// when using icons within buttons you need to include icons as well
+@import '../patterns_icons';
+@include vf-p-icons;
```

Comment:
this is a big mixin, shouldn't we advise including the individual icon needed?

Reply:
Fair point, but for that we would probably need to improve icons docs themselves, to make sure we give detailed instructions how to include each of the icons.

I'll try to add something quickly